content/blog/2020-12-09-slp/index.md

```diff
@@ -0,0 +1,184 @@
++++
+title = "Exploiting Superword Level Parallelism with Multimedia Instruction Sets"
+[extra]
+bio = """
+ Ankush Rayabhari is a first-year MS student in Computer Science at Cornell who
+ is interested in databases and compilers.
+"""
+[[extra.authors]]
+name = "Ankush Rayabhari"
+latex = true
```

Comment:
I think this goes just in `[extra]`, not `[[extra.authors]]`.

Reply:
The other single author blog posts put it there.